packages/react-dom/src/events/SyntheticEvent.js

```diff
@@ -265,7 +266,7 @@ const ClipboardEventInterface: EventInterfaceType = {
   clipboardData: function(event) {
     return 'clipboardData' in event
       ? event.clipboardData
-      : window.clipboardData;
+      : (window: any).clipboardData;
```

Comment:
Would it be worth typing `window` with at least the things we use on it?

Reply:
I was looking into that but then I realize we were already access it as an `any` type. I can try adding the types manually though (I'm surprised there aren't declarations for them already). This `(window: any)` is here though because window could be undefined when exported from Globals